master/buildbot/data/builds.py

```diff
@@ -53,7 +54,13 @@ def get(self, resultSpec, kwargs):
             bldr = kwargs['builderid']
             num = kwargs['number']
             dbdict = yield self.master.db.builds.getBuildByNumber(bldr, num)
-        defer.returnValue((yield self.db2data(dbdict))
+
+        try:
+            props = yield self.master.db.builds.getBuildProperties(dbdict['id'])
```

Comment:
@tardyp A question, your approach is valid only for collection type, isn't it?

* What about single build?
  * All properties filled by default?
  * Actual mechanism => ie. builds/<bid>/properties ?

In my opinion, *builds/<bid>/properties* from DATA API should be removed, as one is forced to request twice API to aggregate data...

Best would be to get all properties of a build without distinction as it is currently the case for single build request and filterable for collection of builds, don't you think?

Concerning collection, a wildcard, for example ``property='*'`` could allow one to get all properties at once without enumerating all of them...